flink-connectors/flink-connector-filesystem/src/main/java/org/apache/flink/streaming/connectors/fs/StringWriter.java

```diff
@@ -54,13 +59,15 @@ public StringWriter() {
 	 *
 	 * @param charsetName Name of the charset to be used, must be valid input for {@code Charset.forName(charsetName)}
 	 */
-	public StringWriter(String charsetName) {
+	public StringWriter(String charsetName, String rowDelimiter) {
```

Comment:
please update the `@param` java doc.

Secondly, I'm not sure, but isn't `StringWriter` a part of public API and shouldn't we preserve the old constructor?

Reply:
Yes  @pnowojski  @StephanEwen